Makefile

```diff
@@ -0,0 +1,143 @@
+	@echo "  install-systemd   - Install $(service), $(timer)"
+	@echo "  install-cronjob   - Install $(cron)"
+	@echo "  uninstall         - Same as uninstall-bin + uninstall-man"
+	@echo "  uninstall-bin     - Uninstall $(bin)"
+	@echo "  uninstall-cfg     - Uninstall $(cfg)"
+	@echo "  uninstall-man     - Uninstall $(man)"
+	@echo "  uninstall-systemd - Uninstall $(service), $(timer)"
+	@echo "  uninstall-cronjob - Uninstall $(cron)"
+	@echo "  purge             - Same as uninstall + uninstall-cfg"
+	@echo "  man                 Create ./danebot.1"
+	@echo "  clean             - Delete ./danebot.1"
+	@echo "  ls                - Run ls -l for all installed files"
+	@echo
+	@echo 'Before installing, read INSTALL for more details.'
+	@echo 'After installing, read "man danebot" for more details.'
+	@echo
+
+check:
+	@fail=0; \
+	for x in $(prereqs); \
+	do \
+		case "`which $$x 2>&1`" in /*) ;; *) echo "Error: $$x is required but is not installed"; fail=1;; esac; \
+	done; \
+	found=0; \
+	for x in $(prereqs_oneof); \
+	do \
+		case "`which $$x 2>&1`" in /*) found=1;; esac; \
+	done; \
+	if [ $$found = 0 ]; then echo "Error: one of $(prereqs_oneof) is required but neither is installed"; fail=1; fi; \
+	for x in $(prereqs_perl); \
+	do \
+		perldoc $$x >/dev/null 2>&1; \
+		if [ $$? != 0 ]; then echo "Error: Perl module $$x is required but is not installed"; fail=1; fi; \
+	done; \
+	[ $$fail = 0 ] || exit 1
+
+install-bin:
+	@if [ ! -d "$(bindir)" ]; then echo "$(bindir) does not exist (adjust Makefile)"; exit 1; fi
+	cp danebot "$(bin)"
+	chmod 755 "$(bin)"
+
+install-cfg:
+	@if [ ! -d "$(cfgdir)" ]; then echo "$(cfgdir) does not exist (adjust Makefile)"; exit 1; fi
+	@if [ -f "$(cfg)" ]; \
+	then \
+		echo "cp danebot.default $(cfg).dist"; \
+		cp danebot.default "$(cfg).dist"; \
+		chmod 644 "$(cfg).dist"; \
+	else \
+		echo "cp danebot.default $(cfg)"; \
+		cp danebot.default "$(cfg)"; \
+		chmod 644 "$(cfg)"; \
+	fi
+
+install-man: man
+	@if [ ! -d "$(mansectdir)" ]; then echo "$(mansectdir) does not exist (adjust Makefile)"; exit 1; fi
+	cp danebot.1 "$(man)"
+	chmod 644 "$(man)"
+
+man: danebot.1
+
+danebot.1: danebot.1.pod
+	sed 's/C</B</g' $< | pod2man --section='$(mansect)' --center='$(mansectname)' --name=DANEBOT --release=danebot --quotes=none > $@
+
+install: install-bin install-cfg install-man $(install_extra)
+
+uninstall-bin:
+	[ ! -f "$(bin)" ] || rm $(bin)
+
+uninstall-cfg:
+	[ "`echo $(cfg)*`" = "$(cfg)*" ] || rm $(cfg)*
+
+uninstall-man:
+	[ "`echo $(man)*`" = "$(man)*" ] || rm $(man)*
+
+uninstall: uninstall-bin uninstall-man $(uninstall_extra)
+
+purge: uninstall uninstall-cfg
+
+clean:
+	[ ! -f danebot.1 ] || rm danebot.1
+
+install-systemd:
+	@if [ ! -d "$(sysdir)" ]; then echo "$(sysdir) does not exist (adjust Makefile)"; exit 1; fi
+	cp danebot.service danebot.timer "$(sysdir)"
+	chmod 644 "$(service)" "$(timer)"
+
+uninstall-systemd:
+	[ ! -f "$(service)" ] || rm $(service)
+	[ ! -f "$(timer)" ] || rm $(timer)
```

Comment:
Should there also be a target to enable the new systemd service?

Reply:
Sure. It's done.

`make systemd-enable` first checks that the certbot.timer isn't enabled. If it is, it tells the user what to do to stop and disable it. If it isn't, it enables and starts the danebot.timer.

`make systemd-disable` stops and disables the danebot.timer.

Also, `make install-systemd` and `make uninstall-systemd` now both run `systemctl daemon-reload`.

And `make uninstall-systemd` does `make systemd-disable` first.

But `make install-systemd` doesn't also do `make systemd-enable` because the user might want to disable the certbot.timer separately to installing danebot. I didn't want the installation to fail after the files were installed.

The INSTALL file is updated to explain what to do.